src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -136,16 +136,12 @@ public void sendBuildMessage(StepParameters stepParameters) {
         WebhookJobProperty property = (WebhookJobProperty) run.getParent().getProperty(WebhookJobProperty.class);
         if (property == null) {
             String webhookUrl = stepParameters.getWebhookUrl();
-            if (StringUtils.isBlank(webhookUrl)) {
-                listener.getLogger().println("No URL provided");
-                return;
-            }
-            try {
-                new URL(webhookUrl);
+            if (FormUtils.validateUrl(webhookUrl)) {
                 Webhook webhook = new Webhook(webhookUrl);
                 executeWorker(webhook, card);
-            } catch (MalformedURLException e) {
-                listener.getLogger().println("Malformed URL provided");
+            } else {
+                listener.getLogger().println("Malformed URL provided, received: '" + webhookUrl + "'");
+                listener.getLogger().println("Expected valid URL or variable reference");
```

Comment:
I had times when running pipeline that I would have liked the logs to indicate that I typed an URL incorrectly